networkx/algorithms/connectivity/edge_augmentation.py

```diff
@@ -1124,15 +1124,16 @@ def complement_edges(G):
     >>> sorted(complement_edges(G))
     []
     """
+    adj = G._adj
```

Comment:
Instead of introducing the new variable `adj`, could we just replace the instances of `G.adj` with `G._adj`?

Reply:
Some basic profiling suggests that a new variable will help with respect to timing performance. From the fact that the `n C 2` for loop will have `LOAD_FAST` instead of the `LOAD_ATTR` (as @boothby mentioned).
But just the change to `G._adj` is a significant speed up.

```

In [31]: G = nx.erdos_renyi_graph(5000, 0.4)

In [32]: %timeit list(c_old(G)) # using G.adj
21.9 s ± 3.18 s per loop (mean ± std. dev. of 7 runs, 1 loop each)

In [33]: %timeit list(c_current(G)) # new variable + G._adj
2.44 s ± 48.8 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)

In [34]: %timeit list(c_adj(G)) # using G._adj
3.5 s ± 100 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
```

Reply:
Got it, thanks for the clarification - I completely missed that :+1: 

+1 for the new variable then, though it might be worth adding a comment with a brief explanation for future readers of the code

Reply:
The new variable's name might be slightly more transparent as `G_adj = G.adj`,   Or `Gadj` or many other forms I am sure. :}
Thanks for this!

Reply:
Renamed to `G_adj` and added a comment for readability.